assembly/src/main/dist/docker/Dockerfile.amd64

```diff
@@ -1,30 +1,32 @@
 FROM amd64/tomcat:8.5-jre8-alpine
 MAINTAINER Bart Hanssens (bart.hanssens@bosa.fgov.be)
 
-ARG VERSION="2.5.1"
-
 ENV JAVA_OPTS="-Xmx2g"
 ENV CATALINA_OPTS="-Dorg.eclipse.rdf4j.appdata.basedir=/var/rdf4j"
 
-RUN adduser -S tomcat 
+RUN adduser -S tomcat
+
+COPY ignore/rdf4j.zip /tmp/rdf4j.zip
+
+WORKDIR /tmp
 
-RUN cd /tmp && \
-	wget -q "https://www.eclipse.org/downloads/download.php?file=/rdf4j/eclipse-rdf4j-${VERSION}-sdk.zip&r=1" -O /tmp/rdf4j.zip && \
-	unzip -q /tmp/rdf4j.zip && \
-	rm -rf /usr/local/tomcat/webapps/* && \
-	cp /tmp/eclipse-rdf4j-${VERSION}/war/*.war /usr/local/tomcat/webapps && \
-	rm -rf /tmp/eclipse && \
-	rm /tmp/rdf4j.zip && \
-	mkdir -p /var/rdf4j && \
-	chown -R tomcat /var/rdf4j /usr/local/tomcat && \
-	chmod a+x /usr/local/tomcat /usr/local/tomcat/bin /usr/local/tomcat/bin/catalina.sh
+RUN	unzip -q /tmp/rdf4j.zip
+RUN	rm -rf /usr/local/tomcat/webapps/*
+RUN	cp /tmp/eclipse-rdf4j*/war/*.war /usr/local/tomcat/webapps
+RUN	rm -rf /tmp/eclipse
+RUN	rm /tmp/rdf4j.zip
+RUN	mkdir -p /var/rdf4j
+RUN	chown -R tomcat /var/rdf4j /usr/local/tomcat
+RUN	chmod a+x /usr/local/tomcat /usr/local/tomcat/bin /usr/local/tomcat/bin/catalina.sh
 
 COPY web.xml /usr/local/tomcat/conf/web.xml
 
 USER tomcat
 
-VOLUME /var/rdf4j
-VOLUME /usr/local/tomcat/logs
+WORKDIR /usr/local/tomcat/
 
-EXPOSE 8080
+# never got this syntax to work with docker-compose
+#VOLUME /var/rdf4j
+#VOLUME /usr/local/tomcat/logs
```

Comment:
I'm not a docker expert but there are a lot of changes in these files that I am unsure about or that look like they're temporary changes for your own test rig, rather than for our deployment. Can you roll these changes back please? If you think some of these are good general improvements let's create a separate issue and PR for it.

Reply:
All of this just fixes up so that the ARM and normal docker images all build the same way (eg. using the code in your branch instead of downloading a release). We were already doing this for the default docker image, just copied it from there so everything was aligned. I've moved it out and made a separate PR.

Reply:
Thanks :)